ladybug_geometry/geometry2d/polygon.py

```diff
@@ -333,15 +333,21 @@ def remove_colinear_vertices(self, tolerance):
 
     def reverse(self):
         """Get a copy of this polygon where the vertices are reversed."""
-        return Polygon2D(tuple(pt for pt in reversed(self.vertices)))
+        _new_poly = Polygon2D(tuple(pt for pt in reversed(self.vertices)))
+        self._transfer_properties(_new_poly)
+        if self._is_clockwise is not None:
```

Comment:
what will happen is it is None?

Reply:
This is what happens if it is None:
https://github.com/chriswmackey/ladybug-geometry/blob/master/ladybug_geometry/geometry3d/face.py#L429-L430

It will get auto-calculated from the `polygon2d` whenever the property is called. I set up all of the Face3D properties like this except the vertices, plane, boundary and holes.  This way, the object is nice and light when it is initialized and all of the other properties are computed and stored when they are called.

Reply:
Nice.